iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -21,7 +21,7 @@
   <!-- delay sign the assembly for Release build -->
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' AND '$(AZURE_IOT_DONOTSIGN.ToUpper())' != 'TRUE' ">
     <AssemblyOriginatorKeyFile>$(RootDir)\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
-    <DelaySign>true</DelaySign>
+    <PublicSign>true</PublicSign>
```

Comment:
Advice from Azure SDK team to switch over using public signing instead of delay signing